src/engine/input_name.asm

```diff
@@ -1245,32 +1337,47 @@ Func_1aefb:
 	ret nz
 	ld a, [wVisibleCursorTile]
 	bit 4, [hl]
-	jr z, Func_1afa1.asm_6fa4
+	jr z, DrawCursor_DeckNamingScreen
+;	fallthrough
 
-Func_1afa1:
+DrawInvisibleCursor_DeckNamingScreen:
 	ld a, [wInvisibleCursorTile]
-.asm_6fa4
+;	fallthrough
+
+; this function is very similar to 'DrawCursor_PlayerNamingScreen'.
+; input:
+;	a = which tile to draw
+;	[wNamingScreenCursorX] = cursor's x position on the keyboard screen
+;	[wNamingScreenCursorY] = cursor's y position on the keyboard screen
+DrawCursor_DeckNamingScreen:
```

Comment:
If this isn't used outside the scope of this routine, why use it as a global label?